backend/seed_data/units.yaml

```
@@ -6,4 +6,7 @@ units:
   - title: "Linked Lists"
     order: 2
     course_id: 1
-  
\ No newline at end of file
+  
+  - title: "Binary Trees"
+    order: 3
+    course_id: 1
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)